protocol/v2/blockchain/beacon/client.go

```diff
@@ -52,6 +52,7 @@ type signer interface {
 // BeaconNode interface for all beacon duty calls
 type BeaconNode interface {
 	specssv.BeaconNode // spec beacon interface
+	GasLimit() uint64
```

Comment:
this is not the prettiest way to expose `GasLimit`, but its already passed to the client. exposing it directly to the running will require changing more critical path code like the runner creation and options.